spice/template.sp

```diff
@@ -0,0 +1,40 @@
+*
+* Filename: test.sp
+* Author: James E. Stine, Jr.
+* Course: none
+* Credits: none
+* Title: GF180 MCU inverter test
+
+***********************************************************************
+* Parameters and models
+***********************************************************************
+.inc "/home/jstine/Desktop/gf180/mcu/018MCU/MODEL/Hspice/YI-141-SM064/Rev9/design.hspice"

```

Comment:
should that be relative? this reference local path.

Reply:
We modified it reference it locally - sorry about that.  Thanks!